icpc-wf/ansible/roles/mysql_server/tasks/main.yml

```diff
@@ -25,3 +25,25 @@
 
 - name: make sure mysql is restarted
   meta: flush_handlers
+
+- name: create directory to store scripts & database dumps
+  file:
+    path: /home/domjudge/{{ item }}
+    owner: domjudge
+    group: domjudge
+    state: directory
+    mode: 0755
+  loop:
+    - db-dumps
+    - scripts
```

Comment:
When is this actually used? In the next one you have bin, which on my VM is also not in my PATH.

Reply:
lol should be bin indeed. Wasn't it bin until I applied your change?
And my GCloud server and my Poli server both had it in bin. So we should maybe (later) figure out if we should add it there?

Reply:
Yes. I think I introduced the error, makes reviewing so much more fun.